src/main/java/fr/theorozier/webstreamer/display/render/DisplayLayerManager.java

```diff
@@ -69,6 +76,8 @@ protected DisplayLayerNode getNewLayer(Key key) throws OutOfLayerException, Unkn
                 return new DisplayLayerHls(key.uri(), this.res);
             } else if (path.endsWith(".jpeg") || path.endsWith(".jpg") || path.endsWith(".bmp") || path.endsWith(".png")) {
                 return new DisplayLayerImage(key.uri(), this.res);
+            } else if (path.endsWith(".svg")) {
+                return svgMap.getNewLayer(key);
```

Comment:
Here, just return a `new DisplayLayerSVGMap(this.res)`.

Reply:
Resolved in https://github.com/mindstorm38/fabric-webstreamer/pull/28/commits/1c0c2e9fdf6e0900dbf82b5f743fdc32ea4a4e03